action.yml

```diff
@@ -39,12 +39,12 @@ inputs:
   version:
     description: |
       The desired version of the [clang-tools](https://github.com/cpp-linter/clang-tools-pip) to use.
-      Accepted options are strings which can be 18, 17, 16, 15, 14, 13, 12, 11, 10, 9, 8 or 7.
+      Accepted options are strings which can be 19, 18, 17, 16, 15, 14, 13, 12, 11, 10, 9, 8 or 7.
 
       - Set this option to a blank string (`''`) to use the platform's default installed version.
       - This value can also be a path to where the clang tools are installed (if using a custom install location).
     required: false
-    default: 12
+    default: 14
```

Comment:
It would be nice if we could set the default `version` value dynamically based on the runner's OS. In Linux, v14 is the default right now, I think. But in Windows, I think the MSVC build tools expects at least v16 or newer. I have no idea what the default version of clang is used in MacOS's Xcode; I would not be surprised if it aligns with Linux though.